release_management/create_artifacts.py

```diff
@@ -90,15 +95,37 @@ def _stratisd_artifacts(namespace):
     if release_version != namespace.version:
         raise RuntimeError("Version mismatch.")
 
-    make_source_tarball("stratisd", release_version, output_abs_path)
-
-    vendor_tarfile_name = vendor(manifest_abs_path, release_version)
-
-    os.rename(vendor_tarfile_name, os.path.join(output_abs_path, vendor_tarfile_name))
-
-    crate_name = f"stratisd-{release_version}.crate"
-    crate_path = os.path.join("target", "package", crate_name)
-    os.rename(crate_path, os.path.join(output_abs_path, crate_name))
+    if namespace.pre_release_suffix:
+        print(f"Using suffix: {namespace.pre_release_suffix}")
+        release_suffix_version = release_version + namespace.pre_release_suffix
+        make_source_tarball("stratisd", release_suffix_version, output_abs_path)
+        vendor_tarfile_name = vendor(manifest_abs_path, release_version)
```

Comment:
I tried adding a "suffix" parameter to the `vendor()` method in `_utils`, but I still need to move the crate to the proper name with the suffix.  Does that mean that `vendor()` would need to move the crate to the new suffix name as well?

Reply:
I think you only need to get the name of the tarfile that you make from the vendor directory correct. You can try calculating that in the vendor() method. Then the vendor() method will return the tarfile name, but it will have the suffix correctly inserted.